girder/api/v1/user.py

```diff
@@ -196,6 +197,15 @@ def deleteUser(self, user):
         self.model('user').remove(user)
         return {'message': 'Deleted user %s.' % user['login']}
 
+    @access.admin
+    @autoDescribeRoute(
+        Description('Get the number of users.')
+    )
+    def getUsersDetails(self):
+        userModelCollection = self.model('user').collection
```

Comment:
Girder's `Model` exposes its own version of `find()` (which returns a PyMongo cursor), so we should use that, instead of the lower-level `collection.find()`. So, this code can just be:
```
nUsers = self.model('user').find().count()
```